app/controllers/concerns/current_cart.rb

```diff
@@ -1,12 +1,13 @@
 module CurrentCart
-	extend ActiveSupport::Concern
```

Comment:
gives access to extended-in module of methods in Active::Support as class methods.